frontend/pages/journey/[id].vue

```diff
@@ -0,0 +1,441 @@
+<script setup lang="ts">
+import {
+  compareAsc,
+  differenceInDays,
+  format,
+  intervalToDuration,
+} from "date-fns";
+import QRCode from "qrcode";
+import JSConfetti from "js-confetti";
+const route = useRoute();
+const journeyId = route.params.id;
+const qrcode = ref("");
+const duringJourney = ref(false);
+const journeyEnded = ref(false);
+const day = ref(0);
+const tensDays = ref(0);
+const hundredsDays = ref(0);
+const jsConfetti = new JSConfetti();
+
+definePageMeta({
+  middleware: ["sanctum:auth"],
+});
+interface Journey {
+  name: string;
+  invite: string;
+  destination: string;
+  from: string;
+  to: string;
+}
+
+const journeyData = ref({} as Journey);
+
+const client = useSanctumClient();
+await client(`/api/journey/${journeyId}`, {
+  method: "Get",
+  async onResponse({ response }) {
+    journeyData.value = response._data;
+  },
+});
+
+const title = journeyData.value.name;
+useHead({
+  title: `${title} | JourneyPlanner`,
+});
+
+QRCode.toDataURL(journeyData.value.invite, function (err, url) {
+  qrcode.value = url;
+});
+
+const fromDate = new Date(journeyData.value.from);
+const toDate = new Date(journeyData.value.to);
+const currentDate = new Date();
+const days = ref(differenceInDays(fromDate, currentDate));
+const daystoEnd = ref(differenceInDays(toDate, currentDate));
+if (days.value > 0) {
+  day.value = Math.floor(days.value % 10);
+  days.value = days.value / 10;
+  tensDays.value = Math.floor(days.value % 10);
+  days.value = days.value / 10;
+  hundredsDays.value = Math.floor(days.value % 10);
+} else if (days.value <= 0 && daystoEnd.value > 0) {
+  duringJourney.value = true;
+  const journeyEnds = ref(differenceInDays(toDate, currentDate));
+  day.value = Math.floor(journeyEnds.value % 10);
+  journeyEnds.value = journeyEnds.value / 10;
+  tensDays.value = Math.floor(journeyEnds.value % 10);
+  journeyEnds.value = journeyEnds.value / 10;
+  hundredsDays.value = Math.floor(journeyEnds.value % 10);
+} else {
+  journeyEnded.value = true;
+}
+
+const isFlipped = ref(false);
+const flip = () => {
+  isFlipped.value = !isFlipped.value;
+};
+</script>
+
+<template>
+  <div class="flex flex-col font-nunito text-text dark:text-white">
+    <div
+      class="absolute right-0 lg:w-1/3 w-full h-10 flex justify-end items-center font-semibold mt-5"
+    >
+      <NuxtLink to="/dashboard" class="flex items-center">
+        <SvgHome class="w-6 h-6" />
+        <p class="text-xl">Dashboard</p>
+      </NuxtLink>
+      <SvgMenu class="w-10 h-10 mx-10" />
+    </div>
+    <div class="flex flex-wrap h-fit mt-[12vh]">
+      <div class="flex w-full items-center justify-center md:hidden">
+        <div class="group w-5/6 [perspective:1000px]" @click="flip">
+          <div
+            :class="isFlipped ? '[transform:rotateX(180deg)]' : ''"
+            class="relative h-full w-full rounded-2xl shadow-xl transition-all duration-500 [transform-style:preserve-3d]"
```

Comment:
shadow entfernen (war auch nicht in Figma), dann verschwinden auch diese weirden weißen Umrandungen

![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/473aa31f-4c20-4b3c-8f44-0a9000d97517)